### ISSUE TITLE
question: Mining mission - cabling and blocks

### ISSUE THREAD

**Issue by DietmarKrause** [zmaster587/AdvancedRocketry#739]
I understand that after positioning the satellite in orbit (as in the video with optical sensor) i put up the block which receives the data. From there i have to do cables to the obervatory in recent versions, which receives the data and there you select the Asteroid. Now write the data form the choosen asteroid to a chip. This chip reads 0/1000 for every type of data. This seems normal until now. Now i have to put that chip in the astrobody data analyzer in the slot to analyze the data. This multiblock structur seems to need the satellite data too to analyze the chips recordings. 
Now the question: how is the cabling supposed to be? Until now i connected the receiving block with all thee types of data to BOTH the obervatory and the astrobody data analyzer. But that seems not working. I have to break the connection to the obervatory and re-initiate the cables to the astrobody data processor to get the analyze working.
second: how do i get a big asteroid? My observatory reads distance 200 and only small asteroids not really woth mining.....

**Comment by zmaster587** [comment]
The cables don't distribute data equally in its current form, so it's possible that it's trying to fill the observatory first before filling the ABDP.  I have noticed periodic issues where the cables forget what network they're part of, but i think those should be mostly fixed by now.  The cable setup you described seems like it should be valid though.

As for the big asteroid, I think only the small asteroid is included in the default config for the latest release.  I had added it to my test world's config and have kept forgetting to add it until now.  The next version should have it by default, but you'd need to let the config regen again.
